src/d3d11/d3d11_video.cpp

```diff
@@ -1328,8 +1328,8 @@ namespace dxvk {
 
   void D3D11VideoContext::CreateSampler() {
     DxvkSamplerCreateInfo samplerInfo;
-    samplerInfo.magFilter       = VK_FILTER_LINEAR;
-    samplerInfo.minFilter       = VK_FILTER_LINEAR;
+    samplerInfo.magFilter       = VK_FILTER_NEAREST;
```

Comment:
Does setting only `magFilter` to `LINEAR` work any better than having both `LINEAR`?

I'd imagine that `NEAREST` is quite nasty when actually performing any upscaling, but a proper fix that promises to never sample outside the source rect has to be implemented at a shader level and is a bit more complex.

Reply:
Not completely sure what you meant, but changing magFilter from LINEAR to NEAREST is what fixes the color bleed and minFilter has no visible effect. I'll update the MR accordingly.

It's related to some larger changes on the Wine side which are not yet merged in Proton, so to reproduce it you will need to build Proton with this wine branch: https://github.com/rbernon/wine/tree/proton-9.0/new-media-source, and then it should happen in Halo Infinite intro videos.